sycl/test-e2e/ESIMD/regression/math_const_fix_test.cpp

```diff
@@ -221,7 +218,7 @@ int test_atan2(float min, float max, float step) {
       float std_result_atan = std::atan2(input_x, input_y);
 
       if (std::fabs(std_result_atan - vector_output_atan2[iy * size + ix]) >
-          0.0001f) {

```

Comment:
1) The test without fast-math passes with the difference 0.0001f. With the proposed change the test compiled without fast-math might eventually start giving difference 3.0f (which most-likely is due to some error), and such error would not be shown by the updated test.
Maybe this would work better:
#ifdef __FAST_MATH__ <use one tolerance> #else <use 0.0001f tolerance> #endif

2) 3.2f seems like quite a big number. It may be Ok for fast-math and atan depending on the input values, I don't know.
Are you sure the device code computes result correctly taken into account such a big difference?